### PR TITLE
Update use-microsoft-tunnel-ios.md

### PR DESCRIPTION
**Pull request by msbemba**
App no longer available in the app store as the support ended. So removed the statement that talks about installing the same from the app store.

IssuE#https://github.com/MicrosoftDocs/memdocs/issues/3012

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: memdocs/intune/user-help/use-microsoft-tunnel-ios.md [hunk 1]
```diff
@@ -36,12 +36,6 @@ Microsoft Tunnel uses Microsoft Defender for Endpoint as The Microsoft Tunnel cl
 
 The Microsoft Tunnel client app helps you securely and privately connect to your corporate network over a VPN. If your organization requires you to use the app, they already configured a VPN connection for your work account. To connect to the VPN, simply install the app and sign in with your work account. 
 
-## Install Microsoft Tunnel  
-
-Tunnel is available in the App Store. Before heading there, check your device to see if it's already installed.   
-
-If you can’t find Tunnel in the App Store, contact your IT support person for help.  
-
```
Comment (JohanFreelancer9) [inline memdocs/intune/user-help/use-microsoft-tunnel-ios.md:39]:
```suggestion
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by lenewsad** [comment]
#sign-off

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Pull request merged**